### PR TITLE
HTTPS base configuration

### PR DESCRIPTION
**Pull request by carboneater**
Minor edits:
- Replace http:8080 server to https:8443
- Moved configuration options to a configuration file in a separate configuration folder. Sync'd for now, will probably go in .gitignore when we get near production
- Added snakeoil cert & key to speed up transfer to SSL.

### CODE REVIEW

**Comment by dozoisch** [comment]
Is that the prod private certificate?

**Comment by carboneater** [comment]
No, development self-signed (snakeoil) certificates.
Prod certificates are not on my laptop.

**Comment by dozoisch** [comment]
:+1: I was scared for a second that there were prod cert on the repo

**Comment by gcourtemanche** [comment]
Good to merge!

File: config/config.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+var config = {};
+var fs = require('fs');
+
+config.jadeoptions = {
```
Comment (gcourtemanche) [inline config/config.js:4]: remove jadeoptions

**Pull request merged**